content/news/XXXX-XX-XX-dvs-internals-pt1.md

```diff
@@ -0,0 +1,157 @@
+Such a signal is called "time-discrete":
+
+![Time-Discrete Signal]({static}/images/news/signal_time_discrete.svg)
+
+The reason for this is that a time-continuous signal has infinite resolution and would need infinite memory space to save, which is impossible - and also unnecessary, because as long as you take enough samples per second, you can [restore the original signal](https://en.wikipedia.org/wiki/Nyquist%E2%80%93Shannon_sampling_theorem)  without any loss of information.
+
+Since computers can't store numbers with a theoretically infinite number of different values, a digital signal is also value-discrete.
+This means that the amplitude values are mapped to a finite number of different values.
+
+Let's assume we only have space to store 11 distinct values per sample (-1.0, -0.8, -0.6, -0.4, -0.2, 0, 0.2, 0.4, 0.6, 0.8 and 1.0).
+But a sample value may have any value, so if its value would be between two of these values, they have to be rounded (in this case: multiples of 0.2).
+This process is called [Quantization](https://en.wikipedia.org/wiki/Quantization_%28signal_processing%29) and causes some information loss, which means that it [adds noise](https://en.wikipedia.org/wiki/Quantization_%28signal_processing%29#Error) to our signal.
+An example for such a quantized, value-discrete version of the analog signal looks like this:
+
+![Value-Discrete Signal]({static}/images/news/signal_value_discrete.svg)
+
+A more realistic example is the Audio CD, which uses 16 bit per sample value.
+With 16 bits, you can represent 2<sup>16</sup> = 65536 different values.
+
+A *digital* signal is both time-discrete and value-discrete:
+
+![Digital Signal]({static}/images/news/signal_digital.svg)
+
+Note that this is just a very basic introduction in how a digital signal is represented.
+If you're interested in the topic, I recommend watching [Monty Montgomery's "Digital Show & Tell" video](https://xiph.org/video/vid2.shtml), where he demonstrates how digital signals behave in contrast to analog ones and clears up common misconceptions.
+
+You may wonder why the timecode signal in the screenshots looks like a analog signal instead of a "lollypop chart".
+*Digital* vinyl scratch system work with digital signals, not analog ones, right?
+And you'd be right, it's just that most audio tools (like Audacity) show a wave instead of individual samples by default.
+
+## Detecting Zero Crossings
+
+Now that we know the basics of digital audio, we can start thinking about a simple low-latency approach to detect the playback velocity using so-called "zero crossings".
+
+Whenever the wave goes from the positive part to the negative part or vice-versa, it crosses "zero".
+For example, in the timecode signal screenshot I used above, we can see 3 zero crossings:
+
+![Zero crossings in timecode signal]({static}/images/news/timecode-signal-samples.png)
```

Comment:
It may help if you explicitly hightlight the zero crossings in the images somehow so people that don't instantly understand the text can still get an intuitive visual understanding what zero crossings are.

Reply:
done.